### DISCUSSION TITLE
In BTLimiter, what is the purpose of the blackboard?

### DISCUSSION THREAD

**Discussion by dfego** [ThePat02/BehaviourToolkit#80] [Q&A] [answered]
I'm writing my own decorators and in looking at `BTLimiter`, I notice it uses the blackboard with a unique key for that particular instance. How does this work better than simply having a variable in the node to represent this value?

**Comment by ThePat02** [comment, marked as answer]
I think this was mostly done to demonstrate how to do something like a unique key. I don't see any other use or why past-me did this.
